tests/Platforms/SQLitePlatformTest.php

```diff
@@ -304,9 +304,9 @@ public function testCreateTableWithDeferredForeignKeys(): void
                 . '"id" INTEGER NOT NULL, "article" INTEGER NOT NULL, "post" INTEGER NOT NULL'
                 . ', "parent" INTEGER NOT NULL, PRIMARY KEY("id")'
                 . ', CONSTRAINT "FK_8D93D64923A0E66" FOREIGN KEY ("article")'
-                . ' REFERENCES "article" ("id") DEFERRABLE INITIALLY IMMEDIATE'
+                . ' REFERENCES "article" ("id") DEFERRABLE'
```

Comment:
`DEFERRABLE` implies `INITIALLY IMMEDIATE`, we just don't render it for brevity.